### PR TITLE
Create animal table

### PR DESCRIPTION
**Pull request by clin2on3mun**
in this milestone : 

- create a database called vet_clinic.
- create animals table with columns(id, name, date_of_birth, escape_attempts, neutered, weight_kg).
-Insert data in table which is data.sql file.
-create queries in queries.

Result of queries and there queries:
- Find all animals whose name ends in "mon".
Screenshot from 2023-08-14 11-49-21
-List the name of all animals born between 2016 and 2019.
date
-List the name of all animals that are neutered and have less than 3 escape attempts.
Screenshot from 2023-08-14 11-51-02
-List the date of birth of all animals named either "Agumon" or "Pikachu".
Screenshot from 2023-08-14 12-04-36
-List name and escape attempts of animals that weigh more than 10.5kg
Screenshot from 2023-08-14 12-13-32

Find all animals that are neutered.
Screenshot from 2023-08-14 11-51-34
-Find all animals not named Gabumon.
Screenshot from 2023-08-14 11-52-27
-Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
Screenshot from 2023-08-14 11-52-04

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-12-31';
+SELECT name FROM animals WHERE neutered=true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name='Pikachu';
+SELECT * FROM animals WHERE neutered=true;
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+SELECT * FROM animals WHERE name NOT IN('Gabumon');
```
Comment (Gambit142) [inline queries.sql]: - [ ] Great work writing most of the queries. Although you have a screenshot, you forgot to write this query in the `queries.sql` file:

> List names and escape attempts of animals that weigh more than 10.5kg.

Kindly include this query as it is part of the requirements for this milestone. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @clin2on3mun 👋, 

Good job so far!

## Highlights
- Everything that creates or changes the structure of the table was saved in the _**schema.sql**_ file ✔️
- Everything that populates tables with data was saved in the _**data.sql**_ file ✔️ 
- Everything that is a query that helped to analyze data was saved in the queries.sql file ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

- Kindly remove this instruction in your README file as it is a guide for you to create your LICENSE file:

image


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: queries.sql (hunk 1 above)
Comment (clin2on3mun) [reply queries.sql]: yeah i saw that and i added it in queries.sql line 6

Comment (Gambit142) [reply queries.sql]: Hi @clin2on3mun, you just added it. Check the highlighted code that I have for the required change. 👍

**Review by AbelG101 — APPROVED** [review]
# Hi @clin2on3mun 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Animals table is properly created ✔
- Animals data is properly inserted ✔
- Correct queries for each question ✔
- Correct use of Git Flow ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**